app/views/layouts/_header.html.erb

```diff
@@ -82,15 +82,17 @@
           </li>
         <% else %>
         <li class="hidden-sm hidden-md"> <!-- signup -->
-          <a rel="tooltip" title="<%= t('layout._header.become_part_community') %>" href="/signup"><%= t('layout._header.join') %></a>
+          <a rel="tooltip" title="<%= t('layout._header.become_part_community') %>" data-toggle="modal" data-target="#userModal" data-mode="signup">
+            <%= t('layout._header.join') %>
+          </a> 
         </li>
         <% end %>
 
         <li class="dropdown">
           <% unless params[:action] == "register" || params[:action] == "signup" %>
             <% if current_user %>
               <a class="dropdown-toggle" data-toggle="dropdown" href="#">
-                Profile  <i class="fa fa-white fa-user" aria-hidden="true" ></i>
+                 Profile  <i class="fa fa-white fa-caret-down"></i> 
```

Comment:
I guess this is overwriting @oorjitchowdhary's last pr changes. Please make it even with the master

Reply:
Yeah. Please don't resolve conversations yourself. 
https://github.com/publiclab/plots2/pull/4199/files are oorjits changes. 
Please integrate those changes also.